style.css

```diff
@@ -353,3 +353,121 @@ header {
 .languages > li {
   margin: 30px 0;
 }
+
+.form-container {
+  height: 844px;
+  padding: 120px 24px;
+  display: flex;
+  flex-direction: column;
```

Comment:
![image](https://user-images.githubusercontent.com/20563252/121273660-557b7e80-c88e-11eb-97b1-e92b58d874ff.png)

When `flex-direction` is activated for the column, the text is before the form. According to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) they should be side by side (two columns) on pc and one column for mobile devices. I kindly invite you to work on this, this will help the portfolio to have a more friendly UI for pc users.